### ISSUE TITLE
Improving website accessibility

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#4136]
This issue is about improving the accessibility of our website outside, mostly apart from the Explore/Validate pages (covered in #2904). I'm just looking to improve visual contrast, add missing alt text, etc to the site for now.
